Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/repositories/MongoProblemRepository.java

```diff
@@ -111,6 +112,18 @@ public Problem save(Problem problem, Map<LanguageEnv, InputStream> providedCodes
         return mongoTemplate.save(problem);
     }
 
+    @Override
+    public int saveProblemWithTitleAndGetId(String title) {
+        int totalProblemsCurrent = mongoTemplate.findAll(Problem.class).size();
```

Comment:
Can't it only count the elements? If you use `findAll` then our performance will have a large impact.